config/prod/ec2-bootstrap-trigger.yaml

```diff
@@ -0,0 +1,7 @@
+template_path: ec2-bootstrap-trigger.yaml
+stack_name: ec2-bootstrap-trigger
+parameters:
+  SsmParameterName: synapse-bridgedownstream-auth
+  PrivateKey: ec2-bootstrap-trigger
+  CrontabURI: s3://{{ stack_group_config.artifact_bucket_name }}/BridgeDownstream/{{ stack_group_config.latest_version }}/ec2/resources/crontab
+  VPCId: vpc-d6c2a9ab
```

Comment:
What is this `vpc-d6c2a9ab`?  Did you create it yourself?

Reply:
You can view the VPC in the console. I didn't create it myself.

Reply:
I took a look at this, and the VPC is just the default VPC created and it doesn't meet our security standards.  This resource needs to sit behind the Sage VPN.

Reply:
See this: https://github.com/nlpsandbox/nlpsandbox-infra/blob/2155e67a552cb7cdaf9237e0bf889ac3f901ea3c/sceptre/nlpsandbox/templates/nlpsandbox.yaml#L121-L127